flask/README.md

```diff
@@ -0,0 +1,49 @@
+# TalkJS and Flask example
+
+This is an example project for TalkJS's tutorial on [how to build a Flask chat app with TalkJS](https://talkjs.com/resources/flask-chat-app/).
+
+This example demonstrates how to integrate TalkJS with a Python application that uses the Flask framework. There are two projects present inside the repo:
+
+- The `talkjs-flask-python` project contains the Flask backend, which serves a REST endpoint with user data.
+- The `talkjs-frontend` project contains the frontend code, which uses TalkJS's [JavaScript SDK](https://talkjs.com/docs/Reference/JavaScript_Chat_SDK/) to create chats between users.
+
+## Prerequisites
+
+To run this tutorial, you will need:
+
+- A [TalkJS account](https://talkjs.com/dashboard/login)
+- [Python](https://www.oracle.com/uk/java/technologies/downloads/)
```

Comment:
```suggestion
- [Python](https://www.python.org/downloads/)
```
This one was going to a Java download :)

Reply:
Brilliant, thanks - thought there might still be copypaste errors